src/mono/wasm/Wasm.Build.Tests/Blazor/BuildPublishTests.cs

```diff
@@ -111,18 +113,28 @@ public static class MyDllImports
         File.WriteAllText(Path.Combine(_projectDir!, "Pages", "MyDllImport.cs"), myDllImportCs);
 
         AddItemsPropertiesToProject(projectFile, extraItems: @"<NativeFileReference Include=""mylib.cpp"" />");
+        BlazorAddRazorButton("cpp_add", """
```

Comment:
I wonder is `[LibraryImport]` also works well.

Reply:
I can add something for that, or try it out, in a follow up PR.